src/web/modules/services/services.exceptions.ts

```diff
@@ -0,0 +1,50 @@
+import { Request, Response, NextFunction } from 'express'
+
+import * as logger from '../../../lib/logger'
+import { EntityNotFoundError, RESTClientError } from '../../../lib/errors'
+
+const detail = function detail(
+  error: Error,
+  req: Request,
+  res: Response,
+  next: NextFunction
+): void {
+  if (error instanceof RESTClientError) {
+    const restClientError = error as RESTClientError
```

Comment:
As @stephencdaly let me know, the error type here is inferred by the context of this block - you shouldn't need this!